From 039e4e5e6dbc0dde28dbd958078abc15c1c0d8b2 Mon Sep 17 00:00:00 2001
From: graham.gower@gmail.com <graham.gower@gmail.com@e8e0d7a0-c8d9-11dd-a880-a1081c7ac358>
Date: Mon, 15 Nov 2010 22:47:14 +0000
Subject: [PATCH] This is not a bug any longer.

It was fixed long ago, as is clear by the code in question.

git-svn-id: http://opkg.googlecode.com/svn/trunk@578 e8e0d7a0-c8d9-11dd-a880-a1081c7ac358
---
 libopkg/opkg_download.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libopkg/opkg_download.c b/libopkg/opkg_download.c
index e84340a..b1f009f 100644
--- a/libopkg/opkg_download.c
+++ b/libopkg/opkg_download.c
@@ -252,7 +252,7 @@ opkg_download_pkg(pkg_t *pkg, const char *dir)
 
     sprintf_alloc(&url, "%s/%s", pkg->src->value, pkg->filename);
 
-    /* XXX: BUG: The pkg->filename might be something like
+    /* The pkg->filename might be something like
        "../../foo.opk". While this is correct, and exactly what we
        want to use to construct url above, here we actually need to
        use just the filename part, without any directory. */
-- 
1.7.1

